src/olympia/amo/tests/__init__.py

```diff
@@ -855,7 +855,7 @@ def version_factory(file_kw=None, **kw):
     addon_type = getattr(kw.get('addon'), 'type', None)
     min_app_version = kw.pop('min_app_version', '4.0.99')
     max_app_version = kw.pop('max_app_version', '5.0.99')
-    version_str = kw.pop('version', '%.1f' % random.uniform(0, 2))
+    version_str = kw.pop('version', '%.1f' % (random.uniform(0, 99) + 0.1))
```

Comment:
Technically not required, but wanted more different version numbers (and avoid `0.0`).

Reply:
`random.uniform(0.1, 99)`?

Reply:
:rofl: not sure why I did it the way that I did... technically doesn't matter though